packages/ragbits-core/src/ragbits/core/prompt/prompt.py

```diff
@@ -23,6 +24,7 @@ class Prompt(Generic[InputT, OutputT], BasePromptWithParser[OutputT], metaclass=
 
     system_prompt: Optional[str] = None
     user_prompt: str
+    images: Optional[List[bytes]] = None
```

Comment:
I don't think this would be useful - here you define images statistically, which means that they will be the same for every instance of the prompt. I don't believe this solves the usual use case - usually images are part of the _input data_, not the static prompt definition.

That's why the ticket says:

> The Prompt class should include its own implementation of the method, that automatically returns images from chosen fields of the **Prompt's input model** (which fields should probably be configured as a static field in the Prompt class)

The aim is to extract the images from the input data, not having them defined statically as part of the class definition.